pureStorageDriver/README.md

```diff
@@ -3,11 +3,10 @@
 This helm chart installs the Pure Service Orchestrator CSI plugin on a Kubernetes cluster.
 
 ## Important Notes
-1. Starting at version 1.0.0, PSO deploys a datastore replicated across the provided storage backends.
+1. Starting at version 6.0.0, PSO deploys a datastore replicated across the provided storage backends.
 1. Currently, there is **no upgrade supported** from previous versions that do not deploy the datastore.
 1. You **MUST** supply a unique `clusterID` in values.yaml. This was previously called `namespace.pure`.
```

Comment:
Can we mention that `clusterID` length needs to be less than or equal to 22 characters?

Reply:
I can make that tag along in this PR, sure.